src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -3181,6 +3181,12 @@ protected BoundExpression BindObjectCreationExpression(ObjectCreationExpressionS
         {
             var type = BindType(node.Type, diagnostics);
 
+            if (node.Type.Kind() == SyntaxKind.TupleType)
+            {
+                Debug.Assert(node.HasErrors, "new <tuple type> should be a syntax error");
+                return BadExpression(node, LookupResultKind.NotCreatable);
```

Comment:
Ah. I see.  I meant to move this to after the check. Somehow type binding is still there. Will fix.

---
In reply to: [81443846](https://github.com/dotnet/roslyn/pull/14161#discussion_r81443846) [](ancestors = 81443846)